step-strikers-app/step-strikers-app/Classes/Caster.swift

```diff
@@ -47,14 +47,11 @@ class Caster: RPGCharacter {
         targetRef.getDocument { (document, error) in
             if let document = document, document.exists {
                 // get info about target's current weapon
-                let weaponName = document.get("current_weapon") as! String
-                let weaponMap = document.get("weapon_inventory") as! [String:[String:Int]]
-                let condition = weaponMap[weaponName]!["condition"]
-                let useCount = weaponMap[weaponName]!["use_count"]
+                let weapon = document.get("current_weapon") as! String
                 
```

Comment:
Can you add a TODO for me split the weapon into weapon and useCount?

Reply:
Yup yup